webapp/src/main/java/com/box/l10n/mojito/monitoring/QuartzPendingJobsReportingTask.java

```diff
@@ -0,0 +1,64 @@
+package com.box.l10n.mojito.monitoring;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tags;
+
+@Component
+@ConditionalOnProperty(value = "l10n.management.metrics.quartz-metrics.enabled", havingValue = "true")
+@ConditionalOnExpression("'${spring.datasource.url:}'.toLowerCase().contains('mysql')")
+public class QuartzPendingJobsReportingTask {
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    MeterRegistry meterRegistry;
+
+    @Scheduled(fixedRateString = "${l10n.management.metrics.quartz-metrics.execution-rate}")
```

Comment:
I suspect we can't use @ConfigurationProperties since it is an annotation?

Reply:
Correct. The spring docs suggest using this format if a property is required.

Reply:
sad, we don't get auto-complete on properties with this I beleive.